config/opal_config_files.m4

```diff
@@ -4,7 +4,7 @@
 # Copyright (c) 2020      The University of Tennessee and The University
 #                         of Tennessee Research Foundation.  All rights
 #                         reserved.
-# Copyright (c) 2022      Amazon.com, Inc. or its affiliates.  All Rights reserved.
```

Comment:
We usually put years in these copyright. I would rather stick with the current model.

Reply:
I'm not a lawyer but it appears to me that each company has a different copyright notice guidance for different reasons. Amazon no longer requires date in the copyright notice, i.e. "dateless".

Not sure if this would matter at all but I can add it back. Probably need to spend 8 years studying US copyright law to understand who should dictate the format (and whether it matters).

Reply:
FWIW: when we first started this project, the lawyers pointed out that a copyright without a date is legally meaningless as copyrights have a mandatory sunset time, and so the project adopted that policy. The copyright in an open-source project isn't all that meaningful - basically, it protects the contributor by ensuring they retain the right to use that file regardless of what anyone else does with it.

Reply:
Indeed, the Berne Convention stipulates that the duration of the term for copyright protection is the life of the author plus at least 50 years after their death (I think this has been upped to 70 in the USA). We can certainly find this info from the commits log, but as a courtesy to the rest of the community we have so far always updated our corresponding copyrights according to our contributions.